apps/meteor/server/models/raw/EmailInbox.ts

```diff
@@ -12,4 +12,32 @@ export class EmailInboxRaw extends BaseRaw<IEmailInbox> implements IEmailInboxMo
 	protected modelIndexes(): IndexDescription[] {
 		return [{ key: { email: 1 }, unique: true }];
 	}
+
+	async setDisabledById(id: IEmailInbox['_id']): Promise<ModifyResult<IEmailInbox>> {
+		return this.findOneAndUpdate({ _id: id, active: true }, { $set: { active: false } }, { returnDocument: 'after' });
+	}
+
+	async removeById(id: IEmailInbox['_id']): Promise<DeleteResult> {
+		return this.deleteOne({ _id: id });
+	}
+
+	async create(emailInbox: IEmailInbox): Promise<InsertOneResult<IEmailInbox>> {
+		return this.insertOne(emailInbox);
+	}
+
+	async updateById(id: IEmailInbox['_id'], data: UpdateFilter<IEmailInbox>): Promise<ModifyResult<IEmailInbox>> {
+		return this.findOneAndUpdate({ _id: id }, data, { returnDocument: 'after', projection: { _id: 1 } });
+	}
+
+	findActiveEmailInboxes(): FindCursor<IEmailInbox> {
```

Comment:
I think adding `EmailInboxes` is redundant.. you also didn't add the suffix to the other methods as well, so I guess we can get rid of it here..

```suggestion
	findActive(): FindCursor<IEmailInbox> {
```